### PR TITLE
Correlation chart

### PR DESCRIPTION
**Pull request by dangn2**
The function to plot correlation chart between stock price and reddit score. Noted that there are some tickers may show better correlation when use submission score value rather than averaged comment score value.

### CODE REVIEW

File: reddit_scraper.py [hunk 1]
```diff
@@ -24,7 +24,6 @@ def main():
 
     for _, stock in stocks.iterrows():
         print(stock['Symbol'] + " " + stock['Name'])
-        query = stock['Symbol']
```
Comment (Jonathan-Birkey) [inline reddit_scraper.py:27]: is this supposed to be removed? it is called here;
`ps_reddit.search_comments(subreddit=SUBREDDIT, q=query, after=start_epoch, before=end_epoch,
                                          sort_type="score", sort="desc", size=25))`

Comment (dangn2) [reply reddit_scraper.py:27]: Chart function should not touch stock/reddit scrapper. I think I accidentally removed it when resolved conflict on my branch.

Comment (Jonathan-Birkey) [reply reddit_scraper.py:27]: for sure, can you please return this line then the PR looks good to me!

**Review by Jonathan-Birkey — APPROVED** [review]

**Review by Jonathan-Birkey — APPROVED** [review]

File: reddit_scraper.py (hunk 1 above)
Comment (dangn2) [reply reddit_scraper.py]: Returned the line. Also, added two empty lines back to the reddit_scraper.

**Pull request merged**